Jellyfin.Api/Controllers/StartupController.cs

```diff
@@ -0,0 +1,127 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Jellyfin.Api.Constants;
+using Jellyfin.Api.Models.StartupDtos;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Library;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    /// <summary>
+    /// The startup wizard controller.
+    /// </summary>
+    [Authorize(Policy = Policies.FirstTimeSetupOrElevated)]
+    public class StartupController : BaseJellyfinApiController
+    {
+        private readonly IServerConfigurationManager _config;
+        private readonly IUserManager _userManager;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="StartupController" /> class.
+        /// </summary>
+        /// <param name="config">The server configuration manager.</param>
+        /// <param name="userManager">The user manager.</param>
+        public StartupController(IServerConfigurationManager config, IUserManager userManager)
+        {
+            _config = config;
+            _userManager = userManager;
+        }
+
+        /// <summary>
+        /// Api endpoint for completing the startup wizard.
+        /// </summary>
+        [HttpPost("Complete")]
+        public void CompleteWizard()
+        {
+            _config.Configuration.IsStartupWizardCompleted = true;
```

Comment:
Shouldn't this happen at the end of the method, after all settings are saved? So if there are issues saving them, a user can "complete" setup again?

Reply:
Out of scope I think. This pr is porting the service to MVC.